packages/ui/v2/core/Shell.tsx

```diff
@@ -802,7 +802,7 @@ export function ShellMain(props: LayoutProps) {
               <div
                 className={classNames(
                   props.backPath ? "relative" : "fixed right-4 bottom-[75px] z-40 ",
-                  "cta mb-4 flex-shrink-0 sm:relative sm:bottom-auto sm:right-auto"
```

Comment:
This global `cta` was actually the main culprit of messing things up. I also tested other pages where this Shell CTA was used, and couldn't find any occurrence where this seemed necessary.